### DISCUSSION TITLE
The ability to select white emotes you want to sync with Discord.

### DISCUSSION THREAD

**Discussion by BradleyjohnsonTV** [night/betterttv#6320] [Ideas]
The one feature I think that would be super awesome if y'all can add is the ability to select what emotes we want to sync with Discord itself instead of it just randomly picking what emotes to sync with discord. Allow us to be able to select what emotes we want to sync with our Discord community, since Discord is mean and does not give us a whole lot to be able to upload. Thank you.

**Comment by night** [comment]
Closing as a duplicate of #5900. Please add your upvote and any extra detail over there so feedback stays consolidated.
